src/config.py

```diff
@@ -1,4 +1,7 @@
 import os
+from dotenv import load_dotenv
```

Comment:
add this as a (dev) dependency to the poetry file or use something external to python (like shell features) to set up the environment instead